test/scenarios.py

```diff
@@ -10325,6 +10325,11 @@
                             "app_version": "test-framework",
                             "prod": false,
                             "init_duration": ">0"
+                        }""",
+                            """{
+                            "type": "transfer_state",
+                            "protocol_version": "6",
+                            "result": 0
```

Comment:
It appears to be just moved around but is no functional chance?

Reply:
the order might have gotten messed up after some rebasing, but moved it here to follow the same layout - events expected in the order that they should arrive in

also, this is still a draft :D

Reply:
aw shiiiiii :D